Example/openxcframework/Utility.swift

```diff
@@ -9,6 +9,8 @@
 import Foundation
 
 public var errorMSG: String = "Error"
+public var prerecordTrace: String = "Pre-recorded Tracefile"
+public var wifinotConnected: String = "WIFI NOT CONNECTED"
```

Comment:
This variable name needs to use consistent camel case. Please change the name from "wifinotConnected" to "wifiNotConnected"